### PR TITLE
Change saga-sdk-proto fields to be consistent

### PR DESCRIPTION
**Pull request by rudyrusli**
Ticket : https://mythicalgames.atlassian.net/browse/MYTHOS-714

### CODE REVIEW

File: api/user/definition.proto [hunk 1]
```diff
@@ -80,14 +80,12 @@ message GetUsersRequest {
...
-    string oauth_id = 2;
+    string oauth_id = 1;
 }
 
 /* Update User call */
 message UpdateUserRequest {
-    string title_id = 1;
-    string oauth_id = 2;
+    string oauth_id = 1;
     // @exclude TODO: what is being updated here
```
Comment (rudyrusli) [inline api/user/definition.proto:89]: Removed title_id from Create/Update User calls

File: api/user/definition.proto [hunk 2]
```diff
@@ -45,7 +45,7 @@ message NftItem {
     /* Name of the ItemType */
     string item_type_name = 1;
     /* Id of the ItemType */
-    string item_type_id = 2;
+    string game_item_type_id = 2;
     /* Contract address of the NFT */
```
Comment (rudyrusli) [inline api/user/definition.proto:49]: For consistency, Users NftItem.item_type_id should be changed to game_item_type_id.

File: api/user/definition.proto [hunk 3]
```diff
@@ -55,9 +55,9 @@ message FungibleToken {
     /* Name for this Fungible Token */
     string name = 1;
     /* Id of game this is part of */
-    string game_id = 2;
+    string title_id = 2;
     /* Amount of token available */
-    string quantity = 3;
+    string balance = 3;
```
Comment (rudyrusli) [inline api/user/definition.proto:60]: FungibleToken.game_id should be changed to title_id and quantity should be changed to balance.

File: api/myth/definition.proto [hunk 4]
```diff
@@ -52,12 +52,12 @@ message QuoteBuyingMythTokenResponse {
...
 message ConfirmBuyingMythTokenRequest {
     string quote_id = 1;
-    string user_id = 2;
+    string oauth_id = 2;
     saga.api.order.PaymentProviderData payment_provider_data = 3;
 }
 
 message QuoteMythTokenWithdrawalRequest {
-    string user_id = 1;
+    string oauth_id = 1;
```
Comment (rudyrusli) [inline api/myth/definition.proto:60]: myth def uses user_id, but that should be oauth_id to be consistent

File: api/payment/definition.proto [hunk 5]
```diff
@@ -40,10 +40,9 @@ message Address {
     string address_line_1 = 3;
     string address_line_2 = 4;
     string city = 5;
-    string state = 6;
+    string state_iso_alpha_2 = 6;
     string postal_code = 7;
-    string country_name = 8;
-    string country_iso_alpha_2 = 9;
+    string country_iso_alpha_2 = 8;
```
Comment (rudyrusli) [inline api/payment/definition.proto:45]: In Address,  remove country_name, and change state to state_iso_alpha_2

**Review by erik-mythical — APPROVED** [review]
Nice

**Review by axcrest — APPROVED** [review]

**Pull request merged**